components/post_view/message_attachments/action_button/action_button.jsx

```diff
@@ -3,25 +3,47 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
+import {changeOpacity} from 'mattermost-redux/utils/theme_utils';
 
 import Markdown from 'components/markdown';
+import {STATUS_COLORS} from 'utils/constants';
 
 export default class ActionButton extends React.PureComponent {
     static propTypes = {
         action: PropTypes.object.isRequired,
         handleAction: PropTypes.func.isRequired,
         disabled: PropTypes.bool,
+        theme: PropTypes.object.isRequired,
     }
 
     render() {
-        const {action, handleAction, disabled} = this.props;
+        const {action, handleAction, disabled, theme} = this.props;
+        let customButtonStyle;
+
+        if (action.style) {
+            const hexColor =
+                STATUS_COLORS[action.style] ||
```

Comment:
Should we test `theme` before the hard-coded `STATUS_COLORS`? Otherwise, will the local theme ever take precedence?

Reply:
I changed `STATUS_COLORS` to be based on the local theme instead of the default. Now it works like in the mobile app. Hope it's ok :+1: